desc/compute/_stability.py

```diff
@@ -231,3 +231,513 @@ def _magnetic_well(params, transforms, profiles, data, **kwargs):
         0,  # coefficient of limit is V_r / V_rr, rest is finite
     )
     return data
+
+
+@register_compute_fun(
+    name="ideal_ball_gamma1",
+    label="\\gamma^2",
+    units=" ",
+    units_long=" ",
+    description="ideal ballooning growth rate" + "requires data along a field line",
```

Comment:
nitpick but right now this description will show as `ideal ballooning growth raterequires data along a field line`

missing a space btwn the two strings

Reply:
Removed stuff after +